docs/pages/kmodal.vue

```diff
@@ -11,6 +11,65 @@
         text="modals"
       />.
     </DocsPageSection>
+    <DocsPageSection
+      title="Usage"
+      anchor="#usage"
+    >
+      <DocsSubNav
+        :items="[
+          { text: 'Dialog box with description', href: '#dialog-box' },
+          { text: 'Submit form dialog box', href: '#submit-form' },
+          { text: 'Modal with different sizes', href: '#modal-sizes' },
```

Comment:
```suggestion
          { text: 'Modal with predefined sizes', href: '#modal-sizes' },
```
To make clearer the distinction between this section and the new precise size section.